doc/man/8/crushtool.rst

```diff
@@ -94,6 +99,16 @@ documented by the **--help-output** option.
    out of **1024** objects ( **4/1024** ) were mapped to **result size
    == 8** devices only.
 
+.. option:: --show-mappings
+
+   Displays the mappings for a range of PG ids (the bounds of which are
+   ``--min-x``, which defaults to 0, and ``--max-x``, which defaults to 
+   1023) or an individual PG id specified by ``--x``. For example::
```

Comment:
I would just write "Display the mapping of each value. For example:". I don't think it's a good idea to mention PG ids at all in this context.